### PR TITLE
Initial attempt at adding a contact info page

### PR DESCRIPTION
**Pull request by sigmavirus24**
Please update this if you see fit. This is just a quick attempt to get something going so we can drive more people to IRC and the mailing list.

Feel free to update wording, the list of clients. If anyone has recommendations for Windows clients, *please* add them. I haven't used IRC on Windows in forever and can't provide any good recommendations. I suspect mirc would still work well but I haven't used it so I don't know.

### CODE REVIEW

File: source/getting-in-touch.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Using IRC
+=========
+
+IRC_ is an old and durable protocol that allows people to chat in real time.
+There are many ways to begin using IRC but we recommend using Freenode's 
+webchat_ as it is both secure (served over TLS) and should work in any
+browser.
+
+If you outgrow Freenode's webcat, there are a number of other options for
```
Comment (The-Compiler) [inline source/getting-in-touch.rst]: While webcats are awesome as well, you meant webchat :wink:

Comment (sigmavirus24) [inline source/getting-in-touch.rst]: :smile_cat:

**Comment by FichteFoll** [comment]
[hexchat](https://hexchat.github.io/) is cross-platform and works nicely on Windows.

**Comment by IanLee1521** [comment]
LGTM, :+1:

**Pull request merged**